src/bootstrap/src/core/build_steps/llvm.rs

```diff
@@ -1172,7 +1174,7 @@ impl Step for CrtBeginEnd {
 
         let crtbegin_src = builder.src.join("src/llvm-project/compiler-rt/lib/builtins/crtbegin.c");
         let crtend_src = builder.src.join("src/llvm-project/compiler-rt/lib/builtins/crtend.c");
-        if up_to_date(&crtbegin_src, &out_dir.join("crtbegin.o"))
+        if up_to_date(&crtbegin_src, &out_dir.join("crtbeginS.o"))
```

Comment:
Previously, we were also copying "crtbeginS". I wonder how it was working before, or wasn't working at all? 🤔

Reply:
We were copying both crtbegin.o and crtend.o. But for some reason, we were checking crtbegin.o (without S) and crtendS.o (with S). So, it was working fine before, but it seemed odd, and I wondered if there was a good reason for it or just a careless inconsistency.